### PR TITLE
Write MULTPV to INIT-file and fix a few minor issues with multiple MULTPV occurrences

### PR DESCRIPTION
**Pull request by blattms**
This output was missing from the init file previously. Now MULTPV will be output if it is specified in the GRID or EDIT section.

Support for MULTPV in the SCHEDULE section will be added in an upcoming PR. I need to sort out the parallel handling of that.

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this please.

**Comment by blattms** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Very good.  We should merge this when we're satisfied that we don't introduce any problems for the simulations.  I just have one question regarding a slightly different way of writing the unit test so as to exercise more of the parser.

File: tests/test_EclipseIO.cpp [hunk 1]
```diff
@@ -739,3 +739,118 @@ BOOST_AUTO_TEST_CASE(MULTXYZInit)
...
+  100*2000 /
+
+PORO
+  100*0.3 /
+
+MULTPV
+  100*10 /  -- overwritten by the next
+
+MULTPV
+  70*1 10*1.5 20*1 /
```
[the remark this answers is not in the document]
Comment (blattms) [reply tests/test_EclipseIO.cpp:800]: It think this should be supported, I will double check.

It reminds about a peculiarity (bug) that I noticed previously, though:
There was a time when `has(keyword)` would return false if a keyword was not specified completely but only in combination with box

[the remark this answers is not in the document]
Comment (blattms) [reply tests/test_EclipseIO.cpp]: I used a slight different aproach where the boxed don't cover the whole domain and can say that we support this.

**Comment by blattms** [comment]
jenkins build this please

File: tests/test_EclipseIO.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (bska) [reply tests/test_EclipseIO.cpp]: > I used a slight different approach where the boxes don't cover the whole domain and can say that we support this.

Excellent&ndash;I'm happy to hear that!

**Comment by blattms** [comment]
Seems like MULTPV in the EDIT section is rather special. It should only work on values that have been edited there. (Not even sure what that means.). Didn't see that coming... Not sure whether master does this correct.

**Comment by blattms** [comment]
jenkins build this please

**Comment by blattms** [comment]
jenkins build this ignore_extra please

**Comment by blattms** [comment]
jenkins build this ignore_extra please

**Comment by blattms** [comment]
I have removed my so-called fix as it broke the PORV of MULTREGT-01.

**Comment by blattms** [comment]
jenkins build this ignore_extra please

**Comment by blattms** [comment]
jenkins build this ignore_extra please

**Comment by akva2** [comment]
whatever you're drinking, i want some

**Pull request merged**